projects/aca-content/src/lib/components/common/toggle-shared/toggle-shared.component.ts

```diff
@@ -62,7 +62,7 @@ export class ToggleSharedComponent implements OnInit, OnDestroy {
       this.selectionState = selectionState;
 
       this.isShared =
-        (this.selectionState.first && this.selectionState.first.entry && (this.selectionState.first.entry as any).sharedByUser) ||
+        (this.selectionState?.first?.entry && (this.selectionState.first.entry as any).sharedByUser) ||
```

Comment:
this line can be reduced to just (this.selectionState?.first?.entry as any)?.sharedByUser
(no need to repeat this.selectionState.first.entry twice)